### PR TITLE
Unit tests scheduled txs

### PR DESCRIPTION
**Pull request by ssd04**

### CODE REVIEW

File: process/block/baseProcess_test.go [hunk 1]
```diff
@@ -1843,4 +1843,423 @@ func TestBaseProcessor_gasAndFeesDelta(t *testing.T) {
...
+			},
+		}
+
+		retBody, err := blproc.GetScheduledMiniBlocksFromMe(header, body)
+		assert.Nil(t, err)
+		assert.Equal(t, block.MiniBlockSlice{mb2}, retBody)
+	})
+}
+
+func TestBaseProcessor_checkScheduledMiniBlockValidity(t *testing.T) {
```
Comment (SebastianMarian) [inline process/block/baseProcess_test.go:2020]: An additional check should be made after the latest push in the main branch:
```
	if len(scheduledMiniBlocks) > len(headerHandler.GetMiniBlockHeadersHashes()) {
		log.Debug("baseProcessor.checkScheduledMiniBlocksValidity", "num mbs scheduled", len(scheduledMiniBlocks), "num mbs received", len(headerHandler.GetMiniBlockHeadersHashes()))
		return process.ErrScheduledMiniBlocksMismatch
	}
```

File: process/block/baseProcess_test.go [hunk 2]
```diff
@@ -2098,6 +2098,38 @@ func TestBaseProcessor_checkScheduledMiniBlockValidity(t *testing.T) {
 		assert.Equal(t, process.ErrScheduledMiniBlocksMismatch, err)
 	})
 
+	t.Run("num header miniblocks higher than scheduled miniblocks, should fail", func(t *testing.T) {
```
Comment (SebastianMarian) [inline process/block/baseProcess_test.go:2101]: lower than...

Comment (ssd04) [reply process/block/baseProcess_test.go]: ahh.. right :+1:

File: process/block/baseProcess_test.go (hunk 1 above)
Comment (ssd04) [reply process/block/baseProcess_test.go:2020]: added additional check

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**